### ISSUE TITLE
Can logging be disabled?

### ISSUE THREAD

**Issue by 93-infinity** [TheGuardianWolf/ThunderbirdTray#25]
tl;dr Could the creation of and/or the appending to of "log.txt" be disabled?

I manage several computers where I have a folder of portable (non-install) apps synced unidirectionally from my pc. Mainly making keeping apps up to date easier.

To prevent unwanted changes synced back to my pc the sync partership is one way (receive only in syncthing) on the computers in question, meaning my computer has the master version and I can override changes.

Anyway the log.txt file is added to all the time, as well as "in use" while ThunderbirdTray is enabled which makes overriding changes more difficult. And reading through the contents of the log, it isn't relevant to me, my log file mainly shows a lot of reporting that it can't find thunderbird's main window.

Could this be disabled?

**Comment by TheGuardianWolf** [comment]
Yep, I can disable this behaviour tomorrow in another release.

I used it previously to debug a previous issue but there is also a flag to enable full debug messages, and the current non debug mode logs are useless as you say.

**Comment by TheGuardianWolf** [comment]
Done, should be in 1.7.1 coming out soon

**Comment by 93-infinity** [comment]
awesome! thanks.
